src/dcurl.c

```diff
@@ -21,7 +21,7 @@
 #include "pow_avx.h"
 #elif defined(ENABLE_SSE)
 #include "pow_sse.h"
-#else
+#elif defined(ENABLE_GENERIC)
```

Comment:
Is the `elif` really necessary? Once all configurable options are not matched, `pow_c` will be picked.